apps/functions/applications-background-jobs/publish-document/index.js

```diff
@@ -65,13 +75,24 @@ export const index = async (
 
 	context.log(`Making POST request to ${requestUri}`);
 
-	await requestWithApiKey
-		.post(requestUri, {
-			json: {
-				publishedBlobContainer: config.BLOB_PUBLISH_CONTAINER,
-				publishedBlobPath: publishFileName,
-				publishedDate: new Date()
-			}
-		})
-		.json();
+	if (migrationPublishing) {
```

Comment:
Can't `migrationPublishing` be done inline since the only difference is `publishedDate`. Just thinking because if I didnt know the ticket I would expect other differences in the request URI etc but its only one field. So we could do  `...(migrationPublishing && { publishedDate: new Date() })`

Reply:
yeah, I did it this way to clearly separate the migration associated code from production code. That way it makes it clear what needs to be removed once migration is done.